_episodes/01-singularity-gettingstarted.md

```diff
@@ -52,6 +52,12 @@ _Later in this material, when we come to look at building Singularity images our
 
 System administrators will not, generally, install Docker on shared computing platforms such as lab desktops, research clusters or HPC platforms because the design of Docker presents potential security issues for shared platforms with multiple users. Singularity, on the other hand, can be run by end-users entirely within "user space", that is, no special administrative privileges need to be assigned to a user in order for them to run and interact with containers on a platform where Singularity has been installed.
 
+## What is the relationship between Singularity, SingularityCE and Apptainer?
+
+Singularity is open source and was initially developed within the research community. The company [Sylabs](https://sylabs.io/) was founded in 2018 to provide commercial support for Singularity.  In [May 2021](https://sylabs.io/2021/05/singularity-community-edition/), Sylabs "forked" the codebase to create a new project called [SingularityCE]((https://sylabs.io/singularity)) (where CE means "Community Edition").  This in effect marks a common point from which two projects---SingularityCE and Singularity---developed. Sylabs continue to develop both the free, open source SingularityCE and a Pro/Enterprise edition of the software. In November 2021, the original open source Singularity project [renamed itself to Apptainer](https://apptainer.org/news/community-announcement-20211130/) and [joined the Linux Foundation](https://www.linuxfoundation.org/press/press-release/new-linux-foundation-project-accelerates-collaboration-on-container-systems-between-enterprise-and-high-performance-computing-environments).
+
+At the time of writing, Apptainer and Singularity are interchangeable, and you might only need to use the `apptainer` command instead of `singularity`.  This course will continue to refer to Singularity until differences between the projects warrant choosing one project or the other for the course material.
```

Comment:
Maybe we could slightly qualify the statement saying that Apptainer and Singularity are interchangeable and also slightly modify the second part of that sentence? For example:

```suggestion
At the time of writing, in the context of the material covered in this lesson, Apptainer and Singularity are effectively interchangeable. If you are working on a platform that now has Apptainer installed, you might find that the only change you need to make when working through this material is to use the the command `apptainer` instead of `singularity`.  This course will continue to refer to Singularity until differences between the projects warrant choosing one project or the other for the course material.
```